src/main/cpp/src/get_json_object.cu

```diff
@@ -1112,7 +1093,7 @@ std::vector<std::unique_ptr<cudf::column>> get_json_object(
     h_path_data, stream, rmm::mr::get_current_device_resource());
   thrust::uninitialized_fill(
     rmm::exec_policy(stream), d_has_out_of_bound.begin(), d_has_out_of_bound.end(), 0);
-  kernel.exec(*d_input_ptr, d_path_data, stream);
+  kernel_launcher::exec(*d_input_ptr, d_path_data, stream);
```

Comment:
When launching the kernel a second time, the number of paths is different from that number in launching the first time.